types/module/module.go

```diff
@@ -185,13 +185,19 @@ func (bm BasicManager) AddQueryCommands(rootQueryCmd *cobra.Command) {
 // AppModuleGenesis is the standard form for an application module genesis functions
 type AppModuleGenesis interface {
```

Comment:
Does this interface still make sense to keep?

Reply:
probably not